test/functional/services/dashboard/add_panel.js

```diff
@@ -18,17 +18,27 @@
  */
 
 
-export function DashboardAddPanelProvider({ getService, getPageObjects }) {
+export async function DashboardAddPanelProvider({ getService, getPageObjects }) {
   const log = getService('log');
   const retry = getService('retry');
   const testSubjects = getService('testSubjects');
+  const webdriver = await getService('__webdriver__').init();
   const flyout = getService('flyout');
   const PageObjects = getPageObjects(['header', 'common']);
+  const driver = webdriver.driver;
 
   return new class DashboardAddPanel {
+    // Dev-helper to pause the browser so we can inspect what's going on
+    // Not meant for usage in CI since it'll stop all execution
+    async pause() {
+      return driver.executeAsyncScript(`(() => { debugger; })()`);
+    }
+
     async clickOpenAddPanel() {
       log.debug('DashboardAddPanel.clickOpenAddPanel');
       await testSubjects.click('dashboardAddPanelButton');
```

Comment:
I think this was the cause of these tests to fail. The add-viz panel animates out, and so it takes some time to be visible in the DOM.